src/api/java/com/minecolonies/api/colony/buildings/modules/IDefinesBuildingModule.java

```diff
@@ -0,0 +1,23 @@
+package com.minecolonies.api.colony.buildings.modules;
+
+/**
+ * Interface describing the different modules.
+ */
+public interface IDefinesBuildingModule
+{
+    /**
+     * Get the max building level a module allows.
+     * By default this is 5.
+     * @return the max level.
+     */
+    default int getMaxBuildingLevel()
```

Comment:
Nope.
This is up to the module.
Either make both methods have a default value or none.

Reply:
It's because maxBuildinglevel = 5 applies to 99% of our buildings. But the number of workers/inhabitants varies greatly. So it makes sense to have a default here but not in the other.